python/doc/theory/reliability_sensitivity/ranking_pcc.rst

```diff
@@ -1,51 +1,52 @@
 .. _ranking_pcc:
 
-Uncertainty ranking: PCC
-------------------------
+Uncertainty ranking: PCC and PRCC
+---------------------------------
 
-This method deals with analyzing the influence the random vector
+Partial Correlation Coefficients
+deal with analyzing the influence the random vector
 :math:`\vect{X} = \left( X^1,\ldots,X^{n_X} \right)` has on a random
```

Comment:
@jschueller / @mbaudin47 : why do we use here a superscript-based notation for the input variables? Regarding the remainder of the documentation, one should use a subscript (e.g., X_{j}^{(i)}, for the i-th sample of the j-th variable).

Reply:
I guess its a mistake, feel free to change it.

Reply:
OK, done.